packages/wow-ui/src/components/Spinner/BlueSpinner.tsx

```diff
@@ -0,0 +1,29 @@
+import Lottie from "lottie-react";
+import type { CSSProperties } from "react";
+
+import blueSpinner from "@/assets/lottie/blueSpinner.json";
+
+/**
+ * @description 블루 스피너 컴포넌트입니다.
+ *
+ * @param {CSSProperties["width"]} [width] - 스피너의 너비.
+ * @param {CSSProperties["height"]} [height] - 스피너의 높이.
+ */
+export interface BlueSpinnerProps {
+  width?: CSSProperties["width"];
+  height?: CSSProperties["height"];
+}
+
```

Comment:
p3;
라이브러리에서 제공하는 Lottie prop  이 있다면  확장해서 써보는 건 어떨까요?! loop 나 autoplay 같은 건 프로퍼티로 제공해도 좋을 것 같아용

```suggestion
export interface BlueSpinnerProps extends LottieProps {
  width?: CSSProperties["width"];
  height?: CSSProperties["height"];
  loop?: boolean;
  autoplay?: boolean;
}

```

Reply:
아하 굿굿 추가해두겠습니다!